src/modules/extendedCallWindow/assets/arrSearch.ts

```diff
@@ -171,14 +171,14 @@ export default (
         aaoGroupElement.prepend(wrapper);
         import(
             /* webpackChunkName: "modules/extendedCallWindow/components/arrSearchDropdown" */
-            '../components/arrSearch/arrSearchDropdown.vue'
-        ).then(({ default: arrSearchDropdown }) =>
+            '../components/arrSearch/ARRSearchDropdown.vue'
```

Comment:
So, we are using PascalCase instead of camelCase? Should be added to the docs as best practice. And, if available as an eslint rule

Reply:
at least the Vue.js Style Guide strongly recommends PascalCase: https://v2.vuejs.org/v2/style-guide/#Single-file-component-filename-casing-strongly-recommended
but yeah, we can also add it to the docs somewhen.

There is an ESLint Rule but I think we shouldn't enable it before #2479 is completed and all Vue files are transitioned to Composition API. What do you think?

Reply:
Add to docs: asap: yeah, but low priority 
Wait till #2479 is completed before enforcing that eslint rule sounds like a good idea as well.